packages/blueprints-integration/package.json

```diff
@@ -15,18 +15,18 @@
 	},
 	"homepage": "https://github.com/nrkno/tv-automation-server-core/blob/master/packages/blueprints-integration#readme",
 	"scripts": {
-		"build": "rimraf dist && yarn build:main",
-		"build:main": "tsc -p tsconfig.build.json",
-		"lint:raw": "eslint --ext .ts --ext .js --ignore-pattern dist",
-		"lint": "yarn lint:raw .",
-		"unit": "jest",
-		"test": "yarn lint && yarn unit",
-		"watch": "jest --watch",
-		"cov": "jest --coverage; open-cli coverage/lcov-report/index.html",
+		"build": "run -T rimraf dist && run build:main",
```

Comment:
I'm curious, what does `run -T` do?

Reply:
`run ...` is a shorthand for `yarn run ...`.
`-T` means top-level. As we don't have `rimraf` defined as a dependency or dev-dependency of `blueprints-integration`, doing a `run rimraf` doesn't find the tool to execute. But the `-T` flag tells it to find the bin/script at the top level of the workspace, where we have got it defined as a dev-dependency.
It was either this, or add rimraf, tsc, and others as a dev-dependency of every package in the workspace. Luckily I discovered this flag before doing that